python/ray/autoscaler/aws/example.yaml

```diff
@@ -3,11 +3,17 @@ cluster_name: default
 
 # The minimum number of workers nodes to launch in addition to the head
 # node. This number should be >= 0.
-min_workers: 2
+min_workers: 1
 
 # The maximum number of workers nodes to launch in addition to the head
 # node. This takes precedence over min_workers.
-max_workers: 4
+max_workers: 2
+
+# This executes all commands on all nodes in the docker container,
+# and opens all the necessary ports to support the Ray cluster.
+docker:
+    image: "" # e.g., tensorflow/tensorflow:1.5.0-py3
```

Comment:
does empty string mean disabled?

Reply:
yeah

Reply:
comment that empty string means disabled?

Reply:
done